sip/execution_framework_interface/docker_api/client/compose-file/docker-compose.visibility.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.6'
+
+services:
+
+    recv1:
+        image: skasip/ingest_visibilities
+        volumes:
+          - type: bind
+            source: /var/mnt/ceph/fred/
```

Comment:
Perhaps change this to be a valid source path, unless you are testing for the case where an invalid path is given.
I noticed in the unit test `test_create_services` where this compose file is used the service creation fails because of this being an invalid path. 
Perhaps use a directory local to the test folder instead?